src/engine/controls/cuecontrol.cpp

```diff
@@ -702,7 +694,6 @@ void CueControl::hotcueGoto(HotcueControl* pControl, double v) {
 void CueControl::hotcueGotoAndStop(HotcueControl* pControl, double v) {
     if (!v)
         return;
-
```

Comment:
I actually like these empty lines, they help to find the distinguish the boilerplate code from the interesting stuff IMHO.

Reply:
yep, but the lock is part of the boilerplate - if you look at all the functions as a whole now, it is very consistent